### DISCUSSION TITLE
Future API Plans

### DISCUSSION THREAD

**Discussion by TheMysterys** [Noxcrew/mcchampionship-api#4] [Ideas]
Below I'm going to be posting some of ideas for the future of the api. 

*Of course it's up to Noxcrew to decide what they do with the api.*

## MCC (The event)
- Historical Data
  - While this was mentioned my Noxcrew as something they are thinking of. Personally I see this as one of the features they should add first
- Live stats
  - This route will provide the current game, coin multiplier, coins and other live information about the event

## MCCI (MCC Island)
These suggestions are things related to an api for MCC Island which could allow players to make tools for looking up players and game info
- Player stats
  - This route shows information about a player including but not limited to first join and last join date, rank, currency, games won, faction, friends, and other player related information
- Game stats
  - This route shows information about a selected game like leaderboard or highest scores, best times and other things related to each game

I plan to expand this list as MCCI grows and more ideas come to my mind. Feel free to suggest your own ideas in the comments below too since I love seeing what other people think.

**Comment by derNiklaas** [comment]
Historical data was added in v1.2.0
